pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -41,6 +42,12 @@ type cohortResources struct {
 	usage                resources.FlavorResourceQuantities
 }
 
+type testOracle struct{}
```

Comment:
why add this, as opposed to use the real object (or not have an intermediary object at all)?

Reply:
dependency graph - preemption depends on flavor assigner, and refactoring was non-trivial